src/libraries/System.Net.ServerSentEvents/src/System/Net/ServerSentEvents/PooledByteBufferWriter.cs

```diff
@@ -14,13 +14,13 @@ internal sealed class PooledByteBufferWriter : IBufferWriter<byte>, IDisposable
 
         public Memory<byte> GetMemory(int sizeHint = 0)
         {
-            _buffer.EnsureAvailableSpace(sizeHint);
+            _buffer.EnsureAvailableSpace(Math.Max(sizeHint, 1));
```

Comment:
`ArrayBuffer` doesn't have its own minimum. I think the intended implementation was with a `private const int MinimumBufferSize = 256` instead of 1.